src/MakeJsonCommand.php

```diff
@@ -132,12 +132,11 @@ public function __invoke( $args, $assoc_args ) {
 
 					if ( $update_mo_files ) {
 						$file_basename    = basename( $file->getFilename(), '.po' );
-						$destination_file = "{$destination}/{$file_basename}.mo";
+						$destination_file = "$destination/$file_basename.mo";
```

Comment:
Why remove the curly braces syntax here? That has been the syntax we're using across all repos so far...

Reply:
It was an IDE recommendation to remove unneeded curly braces.

Happy to add them back, since I shouldn't have included unrelated changes here anyway.

Is there perhaps a lint rule that could enforce this kinda style?

Reply:
Reverted the changes now